deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+# tasks for raster-foundry.jenkins
+
+- name: Add new Nginx config
```

Comment:
The Nginx packages have things setup so that the actual configuration for a site is kept within `/etc/nginx/sites-available/`, which are then symlinked into `/etc/nginx/sites/enabled/` to be activated. The tasks for that roughly look like:

```yaml
- name: Configure Nginx site
  template: src=jenkins.conf.j2
            dest=/etc/nginx/sites-available/jenkins.conf
  notify:
    - Restart Nginx

- name: Enable Nginx site
  file: src=/etc/nginx/sites-available/jenkins.conf
        dest=/etc/nginx/sites-enabled/jenkins
        state=link
  notify:
    - Restart Nginx
```